scripts/correlateTBN.py

```diff
@@ -226,6 +234,8 @@ def main(args):
                         help='filename to correlate')
     parser.add_argument('-m', '--metadata', type=str, 
                         help='name of SSMIF or metadata tarball file to use for mappings')
+    parser.add_argument('-s', '--lwasv', action='store_true', 
```

Comment:
Oh, that's a problem.  There is already a `-s` as the short version of `--samples` which controls how long the many integrations the correlator should process.  In these cases I think I use `-v` like in the case of  [hdfWaterfall.py](https://github.com/lwa-project/commissioning/blob/master/DRX/HDF5/hdfWaterfall.py#L564).

Reply:
good catch! fixed